t/plugin/http-logger2.t

```diff
@@ -32,6 +32,64 @@ add_block_preprocessor(sub {
         $block->set_value("no_error_log", "[error]");
     }
 
+    my $http_config = $block->http_config // <<_EOC_;
+    server {
+        listen 12001;
+
+        location /http-logger/test {
+            content_by_lua_block {
+                ngx.say("test-http-logger-response")
+            }
+        }
+
+        location /http-logger/center {
+            content_by_lua_block {
+                local function str_split(str, reps)
+                    local str_list = {}
+                    string.gsub(str, '[^' .. reps .. ']+', function(w)
+                        table.insert(str_list, w)
+                    end)
+                    return str_list
+                end
+
+                local args = ngx.req.get_uri_args()
+                local query = args.query or nil
+                ngx.req.read_body()
+                local body = ngx.req.get_body_data()
+
+                if query then
+                    if type(query) == "string" then
+                        query = {query}
+                    end
+
+                    local data, err = require("cjson").decode(body)
```

Comment:
Hi, Why do we need to decode the body?

Reply:
The log format data is JSON, and we need to get the information in the structure.

Reply:
Got it.